packages/routing/src/gp2/projects.ts

```diff
@@ -1,15 +1,26 @@
 import { route, stringParser } from 'typesafe-routes';
 import resource from './resource';
+import { outputDocumentTypeParser } from './working-groups';
 
 const project = (() => {
   const edit = route('/edit', {}, { resource });
   const add = route('/add', {}, {});
   const overview = route('/overview', {}, {});
   const resources = route('/resources', {}, { add, edit });
+
+  const upcoming = route('/past', {}, {});
```

Comment:
Why do we have the upcoming and the past with the same route? Shouldn't be /upcoming?

Reply:
yeah, my mistake